python/pyspark/sql/connect/functions.py

```diff
@@ -2471,6 +2472,13 @@ def udf(
 udf.__doc__ = pysparkfuncs.udf.__doc__
 
 
+def _distributed_sequence_id() -> Column:
```

Comment:
Oh, it supposed to be used to create the default index of the pandas API on Spark in the follow-up PR.

To test this function by applying it to the pandas API on Spark code, it requires several other files also must be modified.

So I separated the current work for review convenience.

Reply:
Or I could add another fixes to the current PR if the current fix looks good so far.

If so, this PR would be more like "initial support for pandas API on Spark" rather than a "Add `_distributed_sequence_id` for distributed-sequence index.". 🙂

Reply:
I think you don't have to add this here in this PR. Just directly test Column(DistrivutrdSequenceID())?

Reply:
Ah, I got it.

Reply:
Applied the comments. Thanks!